project/src/components/property-list/property-list.tsx

```diff
@@ -0,0 +1,58 @@
+import {Offer} from '../../types/offer';
+import {upperFirstLetter, calculateRatingWidth} from '../../utils';
+
+type PropertyListProps = {
+  offer: Offer;
+};
+
+export default function PropertyList({offer}: PropertyListProps): JSX.Element {
+  return (
+    <>
+      {offer.isPremium &&
+      <div className="property__mark">
+        <span>Premium</span>
+      </div>}
+
+      <div className="property__name-wrapper">
+        <h1 className="property__name">{offer.title}</h1>
+      </div>
+
+      <div className="property__rating rating">
+        <div className="property__stars rating__stars">
+          <span style={{width: calculateRatingWidth(offer.rating)}}></span>
+          <span className="visually-hidden">Rating</span>
+        </div>
+        <span className="property__rating-value rating__value">{offer.rating}</span>
+      </div>
+
+      <ul className="property__features">
+        <li className="property__feature property__feature--entire">
+          {upperFirstLetter(offer.type)}
+        </li>
+        <li className="property__feature property__feature--bedrooms">
+          {`${offer.bedrooms} Bedrooms`}
```

Comment:
Нет нужды конкатенировать в jsx
```suggestion
          {offer.bedrooms} Bedrooms
```